src/app/progress/BountyGuide.tsx

```diff
@@ -97,20 +113,28 @@ export default function BountyGuide({
       if (info) {
         for (const key in info) {
           for (const value of info[key]) {
-            mapped[key][value] ||= [];
-            mapped[key][value].push(i);
+            (mapped[key][value] ??= []).push(i);
+          }
+        }
+        if (i.pursuit) {
+          for (const reward of i.pursuit.rewards) {
+            if (rewardAllowList.includes(reward.itemHash)) {
+              (mapped.Reward[reward.itemHash] ??= []).push(i);
+            }
           }
         }
       }
     }
   }
 
-  const flattened = Object.entries(mapped).flatMap(([type, mapping]) =>
-    Object.entries(mapping).map(([value, bounties]) => ({
-      type: type as DefType,
-      value: parseInt(value, 10),
-      bounties,
-    }))
+  const flattened = _.sortBy(
+    Object.entries(mapped).flatMap(([type, mapping]) =>
+      Object.entries(mapping).map(([value, bounties]) => ({
+        type: type as DefType,
+        value: parseInt(value, 10),
+        bounties,
+      }))
+    )
```

Comment:
This doesn't seem to be sorting by anything?

Reply:
Ah, just a leftover